var/spack/repos/builtin/packages/dsqss/package.py

```diff
@@ -45,3 +45,23 @@ def cmake_args(self):
         ]
 
         return args
+
+    def test(self):
+        test01 = find(self.prefix.share, '01_spindimer')[0]
+        copy(join_path(test01, 'std.toml'), '.')
+        # prepare
+        pythonexe = self.spec['python'].command.path
+        opts = [self.spec.prefix.bin.dla_pre, 'std.toml']
```

Comment:
This file -- `dla_pre` -- does not appear to be present when I install the package using `spack install dsqss +mpi`.  (I cannot build locally without `+mpi` due to `numactl` configuration errors for *any* of its versions.)

Reply:
`dla_pre` is Python script wrapper.
The make process of DSQSS downloads python files such as https://bootstrap.pypa.io/get-pip.py and build dla_pre.
So please try to export `http_proxy` and `https_proxy`

Reply:
The package installation and test processes should be self contained.  By that I mean the user should be able to run:

```
$ spack install dsqss
$ spack test run dsqss
```

without having to do anything beyond setting up the Spack environment.

Reply:
Please share `spack-build-out.txt`.
Is there error like this?

> WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x40003989ce50>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/pip/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x4000398b9ca0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/pip/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x4000398b9fd0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/pip/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x4000398b9c70>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/pip/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x40003953e100>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/pip/
ERROR: Could not find a version that satisfies the requirement pip (from versions: none)
ERROR: No matching distribution found for pip